### ISSUE TITLE
getting started

### ISSUE THREAD

**Issue by antonkulaga** [valeriyapride777/open_neyrokod#5]
You need a getting started section to explain the steps needed to compile it.
Do I get it right that now it supports Linux?

**Comment by keremet** [comment]
> You need a getting started section to explain the steps needed to compile it.
> Do I get it right that now it supports Linux?

Yes, Linux is supported. Build instructions are here https://github.com/keremet/open_neyrokod/blob/master/BUILD.md

**Comment by highcat** [comment]
Linux was supported a long time ago. BUILD.md was mostly written by me. Newer Qt versions should work too, just try the latest stable version.
